src/taskgraph/generator.py

```diff
@@ -272,7 +272,7 @@ def _run(self):
         # Always add legacy target tasks method until we deprecate that API.
         if "target_tasks_method" not in filters:
             filters.insert(0, "target_tasks_method")
-        filters = [filter_tasks.filter_task_functions[f] for f in filters]
+        filters = [filter_tasks.filter_task_functions[f] for f in filters if f]
```

Comment:
Do you have a traceback of when this caused an error? The `if f` should only be checking that the filter name isn't `None` or an empty string or something. I'm not sure why that would ever happen.

Reply:
this change seems like we can remove

Reply:
Yeah, just from reading the Python I'm pretty sure it'll be a no-op. If `f` is ever a false-y value, then it would cause an exception.. which is desired because it should never be a false-y value. Let's remove this.